slycot/analysis.py

```diff
@@ -679,11 +660,11 @@ def ab09ad(dico,job,equil,n,m,p,A,B,C,nr=None,tol=0,ldwork=None):
     else:
         ordsel = 'F'
     if dico != 'C' and dico != 'D':
-        raise ValueError('Parameter dico had an illegal value')
+        raise SlycotParameterError('Parameter dico had an illegal value', -1)
```

Comment:
Here you are essentially inventing new info numbers. What will they be used for?

Reply:
Ah no, I see, they correspond to the SLICOT parameter number. OK, ignore my remark.